superset/migrations/shared/migrate_viz/base.py

```diff
@@ -68,22 +69,51 @@ def _migrate(self) -> None:
 
             if key in self.rename_keys:
                 rv_data[self.rename_keys[key]] = value
+                continue
 
             if key in self.remove_keys:
                 continue
 
             rv_data[key] = value
 
+        generic_chart_axes = is_feature_enabled("GENERIC_CHART_AXES")
+        if generic_chart_axes:
```

Comment:
nit: as we're not reusing `generic_chart_axes, maybe this would be cleaner:
```suggestion
        if is_feature_enabled("GENERIC_CHART_AXES"):
```